cmd/collector/app/collector.go

```diff
@@ -165,8 +168,12 @@ func (c *Collector) Close() error {
 		c.logger.Error("failed to close span processor.", zap.Error(err))
 	}
 
-	if err := c.tlsCloser.Close(); err != nil {
-		c.logger.Error("failed to close TLS certificate watcher", zap.Error(err))
+	if err := c.tlsGRPCCertWatcherCloser.Close(); err != nil {
+		c.logger.Error("failed to close gRPC TLS certificate watcher", zap.Error(err))
+	}
+
+	if err := c.tlsHTTPCertWatcherCloser.Close(); err != nil {
+		c.logger.Error("failed to close HTTP TLS certificate watcher", zap.Error(err))
```

Comment:
@yurishkuro  According to [codecov report](https://codecov.io/gh/jaegertracing/jaeger/pull/2798/diff
) , only these errors are not covered. The rest are covered. I am not sure how to simulate this error.  Please let me know if you have a way to do so.

Reply:
I suggest replacing these with 
```
// watchers actually never return errors from Close
_ = c.tlsGRPCCertWatcherCloser.Close()
_ = c.tlsHTTPCertWatcherCloser.Close()
```